apis/user-java/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:8-jdk-alpine
```

Comment:
Can we use a newer image and I think we only need the JRE not JDK, yes?  i.e. what about `openjdk:10-jre-slim`